tool_quality_checks/exporter.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+""" 
+Purpose to load postgres checks to geopackage
+"""
+import os
+import logging
+from osgeo import ogr
+from OASDGLDatachecker.tool_quality_checks.copy_ogr_layer import (
+    copy2ogr,
+    set_ogr_connection_pg_database,
+)
+
+logger = logging.getLogger(__name__)
+ogr.UseExceptions()
+OUR_DIR = os.path.dirname(__file__)
+DRIVER_OGR_GPKG = ogr.GetDriverByName("GPKG")
+
+
+def export_checks_from_db_to_gpkg(settings):
+    """
+    Export check files with sewerage background layers into a geopackage
+    """
+
+    # Create output geopackage datasource
+    if os.path.isfile(settings.gpkg_output_layer):
+        os.remove(settings.gpkg_output_layer)
+    gpkg_out_source = None
```

Comment:
Achter deze `= None` zit waarschijnlijk ook een specifieke reden? Even in commentaar (op dezelfde regel) kort erachter zetten.